public/secure-index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en-US">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <!-- For sharing & search engines -->
+  <meta property="og:type" content="website" />
+  <meta property="og:image" content="https://advocacy.mozilla.org/assets/01_SoftwareUpdateImage-Facebook.png" />
+  <meta property="og:image:width" content="1400" />
+  <meta property="og:image:height" content="1400" />
+  <meta property="og:site_name" content="Learn about encryption - Mozilla Advocacy">
+  <meta property="og:determiner" content="" />
+  <meta property="og:locale" content="en_US" />
+  <meta property="og:title" content="Update Your Software">
+  <meta property="og:url" content="https://advocacy.mozilla.org/stay-secure/">
+  <meta property="description" content="Keep all that personal data safer: Take a few minutes and update the software on all your devices -- including phones, tablets, and computers. The web will thank you. Update today with @Mozilla’s help." />
+  <meta property="og:description" content="Keep all that personal data safer: Take a few minutes and update the software on all your devices -- including phones, tablets, and computers. The web will thank you. Update today with @Mozilla’s help.">
+  <!-- <meta property="og:description" content="We Are Building a Global Movement to Protect the Free and Open Web Mozilla supports the heroes of the Web — the developers, advocates and ..."> -->
+  <meta name="twitter:card" content="summary_large_image"/>
+  <meta name="twitter:site" content="@mozilla"/>
+  <meta name="twitter:title" content="Update Your Software"/>
+  <meta name="twitter:description" content="Help safeguard privacy+security by updating your software. Update with @mozilla’s help. #iupdated"/>
+  <meta name="twitter:image" content="https://advocacy.mozilla.org/assets/02_SoftwareUpdateImage-Twitter.png"/>
+
+  <!-- Home string needs to be fixed or removed -->
+  <meta name="title" content="Update Your Software">
+  <meta name="description" content="We Are Building a Global Movement to Protect the Free and Open Web Mozilla supports the heroes of the Web — the developers, advocates and ...">
+  <link rel="stylesheet" type="text/css" href="/build/advocacy.css">
+  <link rel="stylesheet" type="text/css" href="//fonts.googleapis.com/css?family=Fira+Sans:300,400,600">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css">
+  <title>Update Your Software</title>
+  <link rel="apple-touch-icon" type="image/png" sizes="180x180" href="/assets/favicon/apple-touch-icon-180x180.png" >
+  <link rel="icon" type="image/png" sizes="196x196" href="/assets/favicon/favicon-196x196.png">
+  <link rel="shortcut icon" href="/assets/favicon.ico">
+</head>
+
+<body>
+  <div id="my-app"></div>
+  <script src="/build/advocacy.js"></script>
+  <script src="//c.shpg.org/352/sp.js"></script>
+</body>
+
+</html>
```

Comment:
Copy of the other index page but with updated og tags specific to the stay secure campaign.